src/app/components/Search/search-utils.ts

```diff
@@ -1,33 +1,25 @@
-import { Layer } from '../../../config'
+import { LoaderFunctionArgs, useLoaderData } from 'react-router-dom'
 import {
   isValidBlockHeight,
   isValidBlockHash,
   isValidTxHash,
   isValidOasisAddress,
   isValidEthAddress,
+  getEvmBech32Address,
 } from '../../utils/helpers'
-import { RouteUtils } from '../../utils/route-utils'
 
-export abstract class SearchUtils {
-  /**
-   * Receives a search term and returns a matching path
-   */
-  static getNavigationPath(searchTerm: string): string | undefined {
-    const blockHeight = validateAndNormalize.blockHeight(searchTerm)
-    const txHash = validateAndNormalize.txHash(searchTerm)
-    const evmAccount = validateAndNormalize.evmAccount(searchTerm)
-    const consensusAccount = validateAndNormalize.consensusAccount(searchTerm)
-    if (blockHeight) return RouteUtils.getBlockRoute(parseInt(blockHeight, 10), Layer.Emerald)
-    if (txHash) return RouteUtils.getTransactionRoute(txHash, Layer.Emerald)
-    if (evmAccount) return RouteUtils.getAccountRoute(evmAccount, Layer.Emerald)
-    if (consensusAccount) return RouteUtils.getAccountRoute(consensusAccount, Layer.Emerald)
-    // TODO: block hash, contract, validator, event
-    return undefined
-  }
+export const searchSuggestionTerms = {
+  suggestedTransaction: 'b1e68ca814d913064bd6b9460efcb64b4c6d07f3b98fa659beed46164398a830',
+  suggestedBlock: '1396255',
+  suggestedAccount: '0xBA504818FdD8D3dBA2Ef8fD9B4F4D5c71aD1d1D3',
 }
 
 export const validateAndNormalize = {
   blockHeight: (searchTerm: string) => {
+    const thousandSeparator = new Intl.NumberFormat().formatToParts(1234).find(p => p.type === 'group')?.value
```

Comment:
This `1234` looks like it's hardcoded.

Reply:
This is intentional. I couldn't find a thousand separator in Intl.NumberFormat.resolvedOptions, so I use >1000 number to find ","

Reply:
From sync: we should filter out everything but 0-9a-z so that copying someone else's localized "1,201,390" would still parse on other localisations